### DISCUSSION TITLE
Replace the native scrollbar

### DISCUSSION THREAD

**Discussion by mnesser** [MurhafSousli/ngx-scrollbar#457] [Q&A]
Is it possible to replace the “main page” scrollbar with the ngx-scrolbar?

I.e in app.component.html?

```
<ng-scrollbar [autoHeightDisabled]="false">
  <router-outlet>
    <app-spinner></app-spinner>
  </router-outlet>
</ng-scrollbar>
```

The above code does not work. And the native scrollbar is displayed. Setting the height of the ngx-scrollbar to 100vh, works somewhat but gives some issues on mobile browsers.

**Comment by MurhafSousli** [comment]
Yes, it is possible! in this case you should turn off the auto-height feature because you know you will set the height manually.
